models/stg_netsuite2.yml

```diff
@@ -199,13 +199,15 @@ models:
       - name: from_subsidiary_id
         description: The subsidiary id which the consolidated exchange rate is from.
       - name: to_currency_id
-        description: The subsidiary id which the consolidated exchange rate is for.
+        description: The currency id which the consolidated exchange rate is for.
       - name: to_subsidiary_id
         description: The subsidiary id which the consolidated exchange rate is for.
       - name: current_rate
         description: The current rate associated with the exchange rate.
       - name: average_rate
         description: The consolidated exchange rates average rate.
+      - name: accounting_book_id
+        description: Reference to the accounting book of the entry.
```

Comment:
Same comment from the src.yml to leverage the docs.md description.

Reply:
updated